### PR TITLE
:lipstick: [#1498] Add missing icons to plan-edit actions view

### PR DESCRIPTION
**Pull request by jiromaykin**
Make the edit-plan/add-Action look like: 
https://projects.invisionapp.com/share/XP134RWRK3MG#/screens/471522253 
Also there is another page that has missing icons (Start nieuwe samenwerking):
https://projects.invisionapp.com/share/XP134RWRK3MG#/screens/471522250 

Note: due to adding extra class I had to change indentation of the templates.

### CODE REVIEW

File: src/open_inwoner/scss/components/Form/DocumentUpload.scss [hunk 1]
```diff
@@ -43,13 +43,18 @@ input[type='file'] {
...
-  flex-direction: column;
-  min-width: var(--mobile-xs-width);
+  display: inline-block;
+  min-width: 100%;
   max-width: 500px;
   padding: 0;
   overflow: hidden;
   cursor: pointer;
+
+  @media (min-width: 768px) {
```
[the remark this answers is not in the document]
Comment (jiromaykin) [reply src/open_inwoner/scss/components/Form/DocumentUpload.scss:53]: @pi-sigma Thanks :-) yes, i will add a new variable - we will need it more for certain designs.

File: src/open_inwoner/templates/pages/profile/actions/edit.html [hunk 2]
```diff
@@ -3,27 +3,29 @@
...
 
 {% block content %}
-<h1 class="h1">
-    {% if object.pk %}
-        {% trans "Wijzig actie" %}
-    {% else %}
-        {% trans "Creeer actie" %}
-    {% endif %}
-</h1>
+    <div class="plan-edit">
```
Comment (pi-sigma) [inline src/open_inwoner/templates/pages/profile/actions/edit.html:6]: Do we need the indent on the first element? It seems that we don't have a consistent policy on this. Sometimes the first element after a `block` tag is indented, sometimes it isn't.

Comment (vaszig) [reply src/open_inwoner/templates/pages/profile/actions/edit.html:6]: The indentation in the whole file looks good to me (for the block content and the div)

**Review by vaszig — COMMENTED** [review]
I think the `expand-more` icon should follow the `onderwerpen` styling and should change direction when we click it, but I am not sure what's the right approach here.

**Comment by jiromaykin** [comment]
> I think the `expand-more` icon should follow the `onderwerpen` styling and should change direction when we click it, but I am not sure what's the right approach here.

@vaszig I disagree slightly, the Onderwerpen is a different type of interface element than a select, same goes for the FAQ accordion;
if we wish to change all Select elements (which isn't in the design) that should be a new issue.

**Comment by vaszig** [comment]
> @vaszig I disagree slightly, the Onderwerpen is a different type of interface element than a select, same goes for the FAQ accordion; if we wish to change all Select elements (which isn't in the design) that should be a new issue.

You know better!It was just a thought and mostly for my better understanding.

**Review by pi-sigma — APPROVED** [review]
Looking good! I've left a few remarks but they are tangential to the PR since they concern issues that exist in the original version as well.

**Pull request merged**